src/screens/QuickUpdateScreen/QuickUpdate.js

```diff
@@ -608,52 +606,52 @@ class QuickUpdate extends Component {
           </Animated.View>
 
           <Carousel
+            ref={(c) => { this.carousel = c; }}
```

Comment:
do we want to add `removeClippedSubviews` to this so we can get @edencakir's fix in the same pr?